src/components/dropdown-menu/DropdownMenu.scss

```diff
@@ -19,4 +19,17 @@
     margin-top: 5px;
     perspective: 500px;
     perspective-origin: center top;
+
+    &.be-is-small,
+    &.be-is-medium {
+        width: 100%;
+        height: 100%;
```

Comment:
Use of relative height requires a parent element to have a fixed height or to have relative height all the way to the root of the document. Should we use absolute positioning or some other solution, instead?